examples/advanced-video-green-screen.html

```diff
@@ -0,0 +1,227 @@
+  const constraints = {
+    video: true,
+  };
+
+  // Get my camera stream (to do some processing before capturing to canvas
+  const myvideo = document.createElement('video');
+  navigator.mediaDevices.getUserMedia(constraints).then((stream) => {
+    myvideo.srcObject = stream;
+    myvideo.play();
+  });
+
+  // ---------- The gui for configuring other clients inputs  -----------------
+  const API = {
+    GreenThresholdIn: 0.02,
+    GreenThresholdOut: 5,
+  };
+
+  // GUI for changing thresshold for green screen
+  const gui = new dat.GUI( { width: 400 });
+
+  const videoInbound = gui.addFolder("Video Inbound");
+  videoInbound.open();
+
+  videoInbound.add( API, 'GreenThresholdIn', 0, 0.7 ).onChange( function () {
+    document.getElementById("screenPlane").setAttribute("networked-video-source-green-screen","GreenThresholdIn",
+            API.GreenThresholdIn);
+
+  } );
+
+  const videoOutbound = gui.addFolder("Video outbound");
+  videoOutbound.open();
+
+  videoOutbound.add(API, "GreenThresholdOut", 0, 255, 1).onChange((val) => {
+    API.GreenThresholdOut = val;
+  });
+
+  gui.open();
+
+  // Process my stream
+  var canvasElement = document.getElementById('video_output_canvas');
+  var canvasElementSecret = document.getElementById('canvas_secret');
+
+  const w = canvasElement.width;
+
+  console.log("w -----> ", w);
+  const h = canvasElement.height;
+  var canvasCtx = canvasElement.getContext('2d');
+  var canvasCtxSecret = canvasElementSecret.getContext('2d');
+
+  var canvasStream;
+  let r,g,b;
+
+  (function loop() {
+            canvasCtxSecret.drawImage(myvideo, 0, 0, w, h);
+            let imageData = canvasCtxSecret.getImageData(0, 0, w, h);
+
+            for (var i = 0; i < imageData.data.length; i += 4) {
+              r = imageData.data[i]; g = imageData.data[i+1]; b = imageData.data[i+2];
+              if (g-r> API.GreenThresholdOut){
+                imageData.data[i] =0;
+                imageData.data[i+1]=255;
+                imageData.data[i+2]=0;
+              }
+            }
+            canvasCtx.putImageData(imageData, 0, 0);
+            setTimeout(loop, 1000 / 30); // drawing at 30fps
+       }
+  )();
+
+  // final processed stream to send
+  canvasStream = canvasElement.captureStream(30);
+
+  // Screen share status
+  let screenEnabled = false;
+
+  // Screen share button element
+  const screenBtnEle = document.getElementById('screen-btn');
+
+  // Define custom schema for syncing avatar color, set by random-color
+  NAF.schemas.add({
+    template: '#avatar-template',
+    components: [
+      'position',
+      'rotation'
+    ]
+  });
+
+  // Called by Networked-Aframe when connected to server
+  function onConnect () {
+    console.log("onConnect", new Date());
+    setTimeout(function(){
+      // This is the processed stream that we want to send
+      // We do not want the default stream camera (takes bandwidth)
+      NAF.connection.adapter.enableCamera(false);
+    },5000);
```

Comment:
Did you try setting `video:false` on networked-scene like I said in https://github.com/networked-aframe/networked-aframe/issues/296#issuecomment-955667234 ?
As far I understand, having `video:true` implies negotiating a video track with the other participants, and setting `enableCamera(false)` afterwards will just make the track inactive. Setting `video:false` will not add the default camera video track from the RTCPeerConnection at all.
I don't have two PC to test that on my local machine. I usually test with both my PC and iPad, but on the iPad the page freeze.

Reply:
I will try that also. I totally forgot it as it has been a long time ago.

Reply:
Yes you are right, it works with video:false